.circleci/config.yml

```diff
@@ -152,6 +175,32 @@ commands:
                 command: issuegenerator ${TEST_RESULTS}
                 when: on_fail
 
+  run_loadtest:
```

Comment:
Do we use _ or - as separator in job names?

Reply:
For commands, we use `_` but jobs use `-`. I tried to follow existing pattern.